docs/src/overview/comparison.md

```diff
@@ -0,0 +1,58 @@
+# Comparison with commercial Solutions
+
+As metal-stack is the foundation to build kubernetes clusters on premise on bare metal, there are several commercial solutions available which offer management of kubernetes.
+In this document we describe the difference of some of the most popular solutions. It´s is not a complete list.
+
+```@contents
+Pages = ["comparison.md"]
+Depth = 5
+```
+
+In this document we want to compare a often cited battle between Kubernetes managed by Gardener on Metal Stack and Openshift running on VMWare.
+
+## Gardener
+
+[Gardener](https://gardener.cloud) is a Kubernetes Cluster manager to organize a fleet of Kubernetes Clusters at Scale. It is designed to scale to thousands of clusters at a variety of IaaS Providers regardles where, in the Cloud or On Premise, virtualized or bare metal.
+It manages not only the creation and deletion of a Kubernetes Cluster, it also takes care of updates and upgrades of kubernetes and the Operating System of the involved Worker Nodes in a automatic Manner. Its designed Cloud Native as it defines Clustes, Workers and all other Components like Pods and Deployments as Kubernetes Resources which are reconciled to the desired state.
+
+## Kubernetes
+
+[Kubernetes](https://kubernetes.io) is the de facto Open Source standard for Container Scheduling and Orchestration in the Datacenter.
+
+## Openshift
+
+A Fork of Kubernetes with proprietary addons, created by RedHat. For all Details see: [https://en.wikipedia.org/wiki/OpenShift](https://en.wikipedia.org/wiki/OpenShift).
+
+## Metal Stack
+
+Is a [IaaS](https://en.wikipedia.org/wiki/Infrastructure_as_a_service) provider for bare metal focused to create Kubernetes Cluster On Premise. Gardener support is built in.
+
+## VMWare
+
+The most used virtualization technology in the Enterprise Datacenters.
+
+| Feature                                  | Gardener on Metal Stack                                                           | Openshift on VMWare                                         |
+|------------------------------------------|-----------------------------------------------------------------------------------|-------------------------------------------------------------|
+| Container Runtime                        | docker, containerd, gvisor                                                        | cri-o                                                       |
+| Host Operating System                    | Ubuntu, Debian                                                                    | RHEL, Fedora-Core                                           |
```

Comment:
We also support custom images, should link to https://docs.metal-stack.io/previews/PR66/overview/os/